docs/Lexicon.md

```diff
@@ -20,3 +62,15 @@
 * NFCI
    No Functional Change Intended
```

Comment:
what's the difference between NFC and NFCI :)? What are the situation you'd use NFCI?

Reply:
I suppose, *technically* when you can prove mathematically that there is no functional change.  Mostly it is when its clearly an algebraic change versus what you expect to have no functional side-effects.  Most people infer that it was the intention if it turns out not to be the case.